app/src/main/res/layout/dashboard_spotlight_concat.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.LinearLayoutCompat xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
any reason you are using `LinearLayoutCompat` and not just `LinearLayout`? what does `LinearLayoutCompat` provide that isn't part of `LinearLayout`?

Reply:
https://developer.android.com/reference/android/support/v7/widget/LinearLayoutCompat.html#pubmethods

It looks like it isn't supported any longer so I am going to switch.